### PR TITLE
Modify the welcoming message

### PR DESCRIPTION
**Pull request by BatoolMM**
Simple PR to modify the welcoming page!
Can we possibly list all useR groups and R-Ladies group on the landing page or everything should be in a post?

### CODE REVIEW

**Comment by BatoolMM** [comment]
The CI is failing - it can't deploy it!

**Comment by Hussain-Alsalman** [comment]
Tags need to be in quotation marks. Refer to [this](https://github.com/ArabR/ArabR-Website/blob/main/CONTRIBUTING.md#how-to-add-a-post) to learn more about the theme parameters

File: content/posts/Welcoming-Messege.en.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: "Welcome to ArabR"
 date: 2022-01-23T18:01:49+03:00
 draft: false
 author: hussain.en
-tags:
+tags: vision
```
Comment (Hussain-Alsalman) [inline content/posts/Welcoming-Messege.en.md:6]: Tags need to be in quotation marks. Refer to [this](https://github.com/ArabR/ArabR-Website/blob/main/CONTRIBUTING.md#how-to-add-a-post) to learn more about the theme parameters

**Review by Hussain-Alsalman — CHANGES_REQUESTED** [review]

File: content/posts/Welcoming-Messege.en.md (hunk 1 above)
Comment (Hussain-Alsalman) [reply content/posts/Welcoming-Messege.en.md]: Sorry I had not test your PR until now. The problem can be fixed by making tags a list. It should look something like this 
```
tags: 
     - "vision"
```
It seems that quotations are optional.

**Comment by alswajiab** [comment]
As Hussain mentioned, you're missing `-` in your commit.
```
tags: 
     - "vision"
```

**Comment by BatoolMM** [comment]
Thank you @alswajiab and @Hussain-Alsalman!

**Pull request merged**